camelot/backends/pdfium_backend.py

```diff
@@ -0,0 +1,18 @@
+try:
+    import pypdfium2 as pdfium
+except Exception as e:
+    pdfium = None
+    pdfium_exc = e
+else:
+    pdfium_exc = None
+
+
+class PdfiumBackend:
+    def convert(self, pdf_path, png_path, resolution=300):
+        if not pdfium:
+            raise OSError(f"pypdfium2 is not available: {pdfium_exc!r}")
```

Comment:
nit: I doubt if OSError is correct here, but that's what the other backends do ATM.